src/Domain/Entity/Promotion.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Domain\Entity;
+
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use BBC\ProgrammesPagesService\Domain\ValueObject\Synopses;
+
+class Promotion
+{
+    /** @var bool */
+    private $cascadesToDescendants;
+
+    /** @var Pid */
+    private $pid;
+
+    /** @var PromotableInterface */
+    private $promotedEntity;
+
+    /** @var Synopses */
+    private $synopses;
+
+    /** @var string */
+    private $title;
+
+    /** @var string */
+    private $url;
+
+    /** @var int */
+    private $weighting;
+
+    public function __construct(
+        bool $cascadesToDescendants,
+        Pid $pid,
+        PromotableInterface $promotedEntity,
+        Synopses $synopses,
+        string $title,
+        string $url,
+        int $weighting
+    ) {
+        $this->cascadesToDescendants = $cascadesToDescendants;
+        $this->pid = $pid;
+        $this->promotedEntity = $promotedEntity;
+        $this->synopses = $synopses;
+        $this->title = $title;
+        $this->url = $url;
+        $this->weighting = $weighting;
+    }
+
+    public function isCascadesToDescendants(): bool
```

Comment:
well, I thought that we can use something like "isPropagated", or "isSuperpromotion (we call them in this way when they are propagated), but "isCascade" is also  familiar to the field we have in database, so some people is familiar and means pretty much the same.....

someone with any thought?